tools/mason/MasonHelp.chpl

```diff
@@ -444,4 +428,4 @@ proc masonDocHelp(){
   writeln("Requires that chpldoc is set up in order to work.");
   writeln("For instructions on setting up chpldoc, please view its documentation.");
   writeln();
-}
+}
```

Comment:
Note the removal of the EOL character in the diff here. Doesn't really matter in this case, but I've seen this bite people in `.good` files. Good to be aware of in the future.